### DISCUSSION TITLE
Talk to our Product Designers! ☎️

### DISCUSSION THREAD

**Discussion by kusum-ijari** [deepgram/community#190] [General help] [answered]
Hello! We're always looking for feedback to improve our product and would love to hear about your experience setting up and using Deepgram. We would really appreciate it if you could spend some time with us in a feedback session. Its ok if you don't have anything specific to share with us, we already have a plan that will help us understand your journey :) And you might get a sneak peak at some items that are still in development!

You can use [this link](https://calendly.com/kusum-ijari/deepgram-user-testing) to set up some time with us or reach out to us directly and we can set something up.  Thank you for helping us make Deepgram even better!

**Comment by jpvajda** [comment, marked as answer]
If you'd like to share any product feedback with us, please join us on our Discord Server.

https://dpgr.am/discord
